src/shared/theme/pattern.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line prettier/prettier
+const svgDotsPattern =
+`<svg width="8" height="8" viewBox="0 0 8 8" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Try please this: `<svg xmlns="http://www.w3.org/2000/svg" width="8" height="8" fill="none"><path fill="#212121" d="M0 0h8v8H0z"/><path fill="gray" d="M6 2h2v2H6zM2 6h2v2H2z"/></svg>` it's from svgo usage

Reply:
Done.